packages/destination-actions/src/destinations/metronome/index.ts

```diff
@@ -0,0 +1,51 @@
+import { defaultValues, DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import sendEvent from './sendEvent'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Metronome',
+  mode: 'cloud',
+
```

Comment:
Matt corrected me, we should have a slug here. 
```suggestion
slug: 'actions-metronome',
```

Reply:
Added